radis/lbl/factory.py

```diff
@@ -528,15 +528,18 @@ def __init__(
         # if optimization is ``'simple'`` or ``'min-RMS'``, or None :
         # Adjust default values of broadening method :
         if isinstance(broadening_method, Default):
-            if optimization in ("simple", "min-RMS") and broadening_method != "voigt":
+            if (
+                optimization in ("simple", "min-RMS")
+                and broadening_method.value != "voigt"
+            ):
                 if self.verbose >= 3:
                     printg(
-                        "LDM algorithm used. Defaulting broadening method from {0} to Voigt".format(
+                        "LDM algorithm used. Defaulting broadening method from {0} to 'voigt'".format(
```

Comment:
I think we can simplify the code : in both cases (DIT/not DIT) we use "voigt" anyway. Why should we even compare / print a message ? 

All of this block could be reduced to 
```
if isinstance(broadening_method, Default):
    broadening_method = broadening_method.value
```
and maybe we could even avoid using Default ?

Reply:
Yes, Default was used for the purpose that we had different `broadening_methods` for DIT and LBL, but now we are using `voigt` as default so there is no need for `Default`.